### DISCUSSION TITLE
Escritório de Contabilidade 2025/2026

### DISCUSSION THREAD

**Discussion by endersonmenezes** [codaqui/institucional#333] [Enquetes]
Estão de acordo com a taxa de 250 R$ por mês para 2025/2026 e continuar com a Aurum Contabilidade?

Mais informações foram passadas em reunião síncrona e os comentários estão disponíveis para dúvidas. Serão considerados votos validos apenas pessoas que estão ativas ou frequentando reuniões.

**Comment by thaifurforo** [comment]
Dúvida: qual o valor atual?

**Reply by ferfabricio** [reply]
Atualmente o contador não cobra nenhum valor, como estamos aumentando a demanda de requisições para ele, sugeriu que iniciar uma cobrança para o próximo ano.
